data/sql/circles.sql

```diff
@@ -0,0 +1,9 @@
+CREATE EXTENSION IF NOT EXISTS pgcrypto;
+
+CREATE TABLE IF NOT EXISTS circles
+(
+    data json NOT NULL,
+    id uuid DEFAULT gen_random_uuid(),
+    happened_at timestamp without time zone DEFAULT NOW(),
```

Comment:
Because these are circles (and not events) a better name for this column would be `created_at`, and a `modified_at` column would also make sense.